src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustToReviewDto.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dto.trust;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+public class TrustToReviewDto extends TrustDataDto{
```

Comment:
Maybe name this TrustDataToReviewDto
That would make it clearer that this is an extension of TrustDataDto

Reply:
:thumbsup: done